kapitan/cli.py

```diff
@@ -320,6 +320,9 @@ def secret_write(args, ref_controller):
                 logger.warning("WARNING: parameters.kapitan.secrets.recipients is deprecated, " +
                     "please use parameters.kapitan.secrets.gpg.recipients")
                 recipients = inv['nodes'][args.target_name]['parameters']['kapitan']['secrets']['recipients']
+        if not recipients:
+            raise KapitanError("No GPG recipients specified. Use --recipients or specify in " +
```

Comment:
`Use --recipients or specify *them* in`

Reply:
You could move it to just after assignment in line 310.

```
recipients = [dict((("name", name),)) for name in args.recipients]

if not recipients:
    raise KapitanError(..)
```

Reply:
the location is correct, it needs to be done after checking the `args.target_name`. Will fix the typo in print